herald/models.py

```diff
@@ -35,6 +36,7 @@ class SentNotification(models.Model):
     status = models.PositiveSmallIntegerField(choices=STATUSES, default=STATUS_PENDING)
     notification_class = models.CharField(max_length=255)
     error_message = models.CharField(max_length=255, null=True, blank=True)
+    attachments = models.TextField(null=True)
```

Comment:
This should be `blank=True` as well I think. Or the user won't be able to save it in the admin without putting something in. I think...